wire-ios-request-strategy/Sources/Request Strategies/Conversation/ConversationEventProcessor.swift

```diff
@@ -57,6 +57,7 @@ public class ConversationEventProcessor: NSObject, ConversationEventProcessorPro
     // MARK: - Methods
 
     public func processPayload(_ payload: ZMTransportData) {
+        // TODO: [F] should add UUID?
```

Comment:
Assuming this is from some response and not from pulling events or getting it through the push channel, then it's probably fine to init a uuid.